catalystwan/models/configuration/feature_profile/converters/feature_template/aaa.py

```diff
@@ -0,0 +1,30 @@
+from catalystwan.models.configuration.feature_profile.sdwan.system import AAA
+
+
+class AAATemplateConverter:
+    @staticmethod
+    def create_parcel(name: str, description: str, template_values: dict) -> AAA:
+        """
+        Creates an AAA object based on the provided template values.
+
+        Returns:
+            AAA: An AAA object with the provided template values.
+        """
+        template_values["name"] = name
+        template_values["description"] = description
+
+        delete_properties = (
+            "radius_client",
+            "radius_trustsec_group",
+            "rda_server_key",
+            "domain_stripping",
+            "auth_type",
+            "port",
+            "cts_auth_list",
+        )
+
+        for prop in delete_properties:
+            if template_values.get(prop) is not None:
+                del template_values[prop]
```

Comment:
Input dict template_values is being modified so it cannot be used in the same manner again.
I believe this was not intended. I suggest using deepcopy and pass it to AAA